src/components/utils/include/utils/log_message_loop_thread.h

```diff
@@ -52,9 +54,24 @@ typedef struct {
 } LogMessage;
 
 typedef std::queue<LogMessage> LogMessageQueue;
+class LogMessageLoopThreadTemplate
+    : public threads::MessageLoopThread<LogMessageQueue> {
+ public:
+  // Places a message to the therad's queue. Thread-safe.
+  void PostMessage(const Message& message) OVERRIDE {
+    threads::MessageLoopThread<LogMessageQueue>::PostMessage(message);
+
+    // LOG_MESSAGE_QUEUE_SIZE defined in Utils CMakeLists
+    if (GetMessageQueueSize() > LOG_MESSAGE_QUEUE_SIZE) {
+      // TODO (AKutsan): Avoid stopping thread if message queue is full.
+      // Prefere Handle multiple messages from Message loop thread
+      WaitDumpQueue();
```

Comment:
@LuxoftAKutsan, inconsistence of the following message need to be forced logged.
I assume, that logging in case of over-limitation logging sounds strange, but I is mandatory to save a state that all following messages are not trusted any more.
As a one case of implementation - we can log this fatal ones per SDL life_cycle.